packages/devextreme/js/__internal/grids/grid_core/virtual_scrolling/m_virtual_scrolling.ts

```diff
@@ -1789,10 +1789,15 @@ export const rowsView = (Base: ModuleType<RowsView>) => class VirtualScrollingRo
     super.setLoading.call(this, isLoading, messageText);
   }
 
+  private isRowDragging() {
+    return this.component.option('isDragging');
+  }
```

Comment:
Can we rename this method to 'isDraggableGrid' and move it to another place? Doesn't seem correct to me to place this kind of method in VirtualScrolling module